src/pivotal-ui/javascripts/pivotal-ui.js

```diff
@@ -7,3 +7,4 @@ require('pui-prismjs');
 require('./scale')();
 require('@npmcorp/pui-react-back-to-top/jquery-plugin');
 require('@npmcorp/pui-css-drop-down-menu');
+require('../components/forms');
```

Comment:
Ya know, I think it's better to have it require like this (and the drop-down-menu should be changed to match thusly in a chore). That way, when you're making changes locally, they'll show up right away in the styleguide.